gravitee-am-ui/src/app/domain/applications/application/overview/overview.component.html

```diff
@@ -112,7 +112,9 @@ <h2>Initiate the Login Flow</h2>
 curl -X POST \
               {{ baseUrl }}/oauth/token \
   -H 'Content-Type: application/x-www-form-urlencoded' \
-  -d 'grant_type=client_credentials&client_id={{ clientId }}&client_secret=&lt;<i>client_secret</i>&gt;'</pre
+  -d 'grant_type=authorization_code&code=|code|&redirect_uri={{ redirectUri }}&client_id={{
+                clientId
+              }}&client_secret=&lt;<i>client_secret</i>&gt;{{ getCodeVerifier() }}'</pre
```

Comment:
though: 

should it be one way to say that there is a variable to replace?
if we have for client secret: &lt;<i>client_secret</i>&gt;
then the same for code: &lt;<i>code</i>&gt;

What do you think?